### PR TITLE
update installation instructions in README

### PR DESCRIPTION
**Pull request by dokkaebi**
Had to make a few changes to get this working on scala 2.12 with kamon 1.1.2. Updated the readme with what I ended up using. Note - I did not test 0.0.4 at all, I just noticed it is listed in the jitpack repo so I assume it works with 2.11.

### CODE REVIEW

**Comment by jtjeferreira** [comment]
Jitpatck is a bit weird, but 0.0.4 should work with 2.12. Just try to use it in your build and it should trigger publishing...

**Comment by dokkaebi** [comment]
Yep that worked, thanks. Updated.

File: README.md [hunk 1]
```diff
@@ -19,14 +19,14 @@ Supported releases and dependencies are shown below.
...
-|  0.0.1 | 0.12 until 0.16 | stable  | 1.8+ | 2.11, 2.12
+|  0.0.4 | 0.12 until 0.16 | stable  | 1.8+ | 2.11, 2.12
 
 
 To get started with SBT, simply add the following to your `build.sbt` file:
 
 ```scala
 resolvers += "jitpack" at "https://jitpack.io"
-libraryDependencies += "com.github.jtjeferreira" % "kamon-mongo" % "0.0.1"
+libraryDependencies += "com.github.jtjeferreira" %% "kamon-mongo" % "0.0.3"
```
Comment (jtjeferreira) [inline README.md]: Can you change to 0.0.4 here?

Comment (dokkaebi) [reply README.md]: Oops. Thanks for your vigilance. And for the library!

**Review by jtjeferreira — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jtjeferreira) [reply README.md]: Thanks for the contribution, very cool to know this is useful for someone else...

**Pull request merged**